### PR TITLE
Allow file and directory names to contain percent character

### PR DESCRIPTION
**Pull request by wqweto**
This is a quick fix for #1137

`EscapePound` could be expanded to full blown url_encode probably using URL from `net/url` but I'm not sure how this will affect performance as it's a fairly frequently used function in templates.

Speaking of templates, every href attribute has to use `EscapePound` on user generated content i.e. various names (repo, user, directory, file) or risk invalid links.

### CODE REVIEW

File: templates/repo/home.tmpl [hunk 1]
```diff
@@ -48,7 +48,7 @@
                     {{ $l := Subtract $n 1}}
                     {{range $i, $v := .Treenames}}
                         {{if eq $i $l}}
-                        <span class="bread">{{EscapePound $v}}</span>
+                        <span class="bread">{{$v}}</span>
```
Comment (Unknwon) [inline templates/repo/home.tmpl]: Why did you remove this one? Is it related to #1137?

Comment (wqweto) [inline templates/repo/home.tmpl]: Yes, instead of `fcfg-eth1%3A1` it was showing filename as `ifcfg-eth1%253A2` i.e. double escaped. Same would happen for filenames with # I suppose, don't know what the original intent was.

Comment (Unknwon) [inline templates/repo/home.tmpl]: Make sense, maybe just was a mistake!

**Comment by Unknwon** [comment]
Thanks!
